src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,103 @@
+import PropTypes from 'prop-types';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMoviesData } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState(null);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {}, [title]);
+
+  const getMovie = () => {
+    getMoviesData(title)
+      .then((foundedMovie) => {
+        if (foundedMovie.Response === 'False') {
+          setError(true);
+          setMovie(null);
+
+          return;
+        }
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+        setMovie({
+          title: foundedMovie.Title,
+          description: foundedMovie.Plot,
+          imgUrl: foundedMovie.Poster,
+          imdbUrl: `https://www.imdb.com/title/${foundedMovie.imdbID}/`,
+          imdbId: foundedMovie.imdbID,
+        });
+        setError(false);
+        setTitle('');
+      });
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classNames(
+                'input', { 'is-danger': error },
+              )}
+              value={title}
+              onChange={(event) => {
+                setTitle(event.target.value);
+                setError(false);
+              }}
+            />
+          </div>
+          {error && (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              className="button is-light"
+              onClick={getMovie}
```

Comment:
is better to give users the opportunity to find a movie with an 'enter' key. use onSubmit for this